src/WormholeEndpointStandalone.sol

```diff
@@ -29,4 +29,18 @@ contract WormholeEndpointStandalone is WormholeEndpoint, EndpointStandalone {
     function pauseEndpoint() external override onlyOwnerOrPauser {
         _pause();
     }
+
+    /// @dev Override the [`transferOwnership`] method from OwnableUpgradeable
+    /// to ensure owner cannot transfer ownership
```

Comment:
"..directly. The owner can still transfer ownership via the Manager. This is to ensure the owner of this contract is in sync with the owner of the Manager"